components/ChannelProfile/ChannelProfile.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { Modal, View, Image, TouchableOpacity } from 'react-native';
+import { Text, Icon, Card, CardItem } from 'native-base';
+import { Font, AppLoading } from "expo";
+import AutoLink from 'react-native-autolink'
+import _ from 'lodash';
```

Comment:
Looks like not all of these are actually being used, so it's probably best to remove them.

eg. Font, AppLoading, lodash